docker-compose.yml

```diff
@@ -30,8 +30,6 @@ services:
       CONCOURSE_ADD_LOCAL_USER: test:test,guest:guest
       CONCOURSE_MAIN_TEAM_LOCAL_USER: test
       CONCOURSE_CLUSTER_NAME: dev
-      CONCOURSE_CLIENT_SECRET: Y29uY291cnNlLXdlYgo=
```

Comment:
Given that `web` has both `tsa` and `tsa` together, and when running `docker-compose` for dev purposes only we have only a single web node, there's no problem in having these secrets being rotate on every restart of `web` - is that right?

Reply:
Nope. Not a problem. These values get threaded through the system at startup, and saved in the db under the dex `client` table. So everything can stay in sync.